src/__experimental__/components/switch/switch.spec.js

```diff
@@ -400,6 +401,16 @@ describe('Switch', () => {
       });
     });
   });
+
+  it('the required prop is passed to the input', () => {
+    const wrapper = render({ required: true, label: 'required' }, mount);
+
+    const input = wrapper.find('input');
+    expect(input.prop('required')).toBe(true);
+
+    const label = wrapper.find(Label);
+    expect(label.prop('isRequired')).toBe(true);
+  });
```

Comment:
two tests in one case

Reply:
Updated test description because the setup is the same.